pyzabbix/sender.py

```diff
@@ -48,8 +48,8 @@ def __init__(self):
         self._total = 0
         self._time = 0
         self._chunk = 0
-
-        self._regex = re.compile(r'processed: (\d*); failed: (\d*); total: (\d*); seconds spent: (\d*\.\d*)')
+        self._regex = re.compile('processed: (\d*); failed: (\d*); total:\
+ (\d*); seconds spent: (\d*\.\d*)')
```

Comment:
I think this would work better:
```python
self._regex = re.compile(r'processed: (\d*); failed: (\d*); total: '
                                       '(\d*); seconds spent: (\d*\.\d*)')
```
or even:

```python
pattern = r'processed: (\d*); failed: (\d*); total: (\d*); seconds spent: (\d*\.\d*)'
self._regex = re.compile(pattern)
```

it should be with `r'string'`